theatre/studio/src/store/index.ts

```diff
@@ -11,6 +11,7 @@ import logger from '@theatre/shared/logger'
 const initialState: StudioState = {
   ahistoric: {
     visibilityState: 'everythingIsVisible',
+    keyframesClipboard: [],
```

Comment:
This is optional, so we don't have to include it in the `initialState`. Let's leave the initial state to reflect how things looked like at theatre.js `0.4.0`